src/injected/selector-map-helper.ts

```diff
@@ -58,6 +60,15 @@ export class SelectorMapHelper {
             );
         }
 
+        if (visualizationType === VisualizationType.AutomatedChecks) {
```

Comment:
I think we could use `AutomatedChecks.key` (from the `automated-checks/assessment.tsx`) instead because that would work for both QuickAssess.

Additionally, we'd have to modify `visualization-state-change-handler.ts` to pass the appropriate store's card selection data.